### PR TITLE
Feature/incremental deploy

### PR DESCRIPTION
**Pull request by liayoo**
- Added deploy_blockchain_incremental_gcp.sh that runs start_node_incremental_gcp.sh
- Incremental deploy script names a directory with the date as a suffix
- Kills & restarts nodes one by one, without resetting the chain
- Fixed address not defined error

### CODE REVIEW

File: deploy_blockchain_incremental_gcp.sh [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+gcloud compute scp --recurse $FILES_FOR_NODE ${NODE_1_TARGET_ADDR}:~/ --project $PROJECT_ID
+printf "\nCopying files to ${NODE_2_TARGET_ADDR}..."
+gcloud compute scp --recurse $FILES_FOR_NODE ${NODE_2_TARGET_ADDR}:~/ --project $PROJECT_ID
+printf "\nCopying files to ${NODE_3_TARGET_ADDR}..."
+gcloud compute scp --recurse $FILES_FOR_NODE ${NODE_3_TARGET_ADDR}:~/ --project $PROJECT_ID
+printf "\nCopying files to ${NODE_4_TARGET_ADDR}..."
+gcloud compute scp --recurse $FILES_FOR_NODE ${NODE_4_TARGET_ADDR}:~/ --project $PROJECT_ID
+
+# ssh into each instance, set up the ubuntu VM instance (ONLY NEEDED FOR THE FIRST TIME)
+if [ "$4" == "--setup" ]; then
```
Comment (platfowner) [inline deploy_blockchain_incremental_gcp.sh:68]: Wow, good to see this! 
Could you add this option to other scripts as well? (e.g. deploy_blockchain_gcp.sh)

File: deploy_blockchain_incremental_gcp.sh [hunk 2]
```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+
+if [ "$#" -lt 3 ]; then
+    echo "Usage: sh deploy_blockchain_incremental_gcp.sh [dev|staging|spring|summer] <GCP Username> <# of Shards> --setup"
```
Comment (platfowner) [inline deploy_blockchain_incremental_gcp.sh:4]: As `--setup` is optional, what about adding brackets around? (i.e. `[--setup]`)

File: deploy_blockchain_incremental_gcp.sh (hunk 1 above)
Comment (liayoo) [reply deploy_blockchain_incremental_gcp.sh:68]: Done! 😃

File: deploy_blockchain_incremental_gcp.sh (hunk 2 above)
Comment (liayoo) [reply deploy_blockchain_incremental_gcp.sh]: Done!

File: start_node_incremental_gcp.sh [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+sudo rm -rf ../ain-blockchain*
+
+# 4. Create a new directory
+sudo mkdir $NEW_DIR_PATH
+sudo chmod 777 $NEW_DIR_PATH
+mv * $NEW_DIR_PATH
+
+# 5. Start a new node process
+cd $NEW_DIR_PATH
+npm install
```
Comment (platfowner) [inline start_node_incremental_gcp.sh:119]: minor: what about defining a separate step for installation>

File: start_node_incremental_gcp.sh [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+while :
+do
+    consensusState=$(curl -X POST -H "Content-Type: application/json" --data "$(generate_post_data)" "http://localhost:8080/json-rpc" | jq -r '.result.result.state')
+    printf "\nconsensusState = ${consensusState}"
+    if [ "$consensusState" == "RUNNING" ]; then
+        printf "\nNode is synced & running!\n\n"
+        break
+    fi
+    ((loopCount++))
+    printf "\nLoop count: ${loopCount}\n"
```
Comment (platfowner) [inline start_node_incremental_gcp.sh:142]: Is it possible to fetch and print the last block number? (e.g. using ain_getRecentBlockNumber())

File: start_node_incremental_gcp.sh (hunk 3 above)
Comment (liayoo) [reply start_node_incremental_gcp.sh:119]: Done!

File: start_node_incremental_gcp.sh (hunk 4 above)
Comment (liayoo) [reply start_node_incremental_gcp.sh:146]: Great idea, done!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by liayoo** [comment]
Thanks for the review! Now merging~

**Pull request merged**